java/bulk-export/bulk-export-core/src/main/java/sleeper/bulkexport/model/BulkExportQueryException.java

```diff
@@ -13,11 +13,22 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package sleeper.bulkexport.starter;
+package sleeper.bulkexport.model;
 
 /**
- * Lambda to start the bulk export job.
+ * A custom exception for when a bulk export goes wrong.
  */
-public class BulkExportStarterLambda {
+public class BulkExportQueryException extends Exception {
```

Comment:
It looks like this class is not used, and can be deleted?

Reply:
I'll remove it